src/app/dim-ui/Select.tsx

```diff
@@ -107,9 +109,12 @@ export default function Select<T>({
   }
 
   if (dropdownWidth !== undefined) {
-    dropdownStyle = {
-      maxWidth: dropdownWidth,
-    };
+    dropdownStyle.maxWidth = dropdownWidth;
+  }
+
+  if (maxDropdownHeight !== undefined) {
+    dropdownStyle.maxHeight = maxDropdownHeight;
+    dropdownStyle.overflowY = 'scroll';
```

Comment:
Let's also add `overscroll-behavior: contain` to the dropdown's stylesheet to prevent that scroll chaining behavior, and move the `overflow-y: scroll` into the stylesheet as well (it can always be applied).

Reply:
Nice, didn't know about this one.